src/views/pages/groups/components/Group2.js

```diff
@@ -0,0 +1,99 @@
+import PropTypes from "prop-types";
+
+// material-ui
+import { useTheme, styled } from "@mui/material/styles";
+import { Avatar, Box, List, ListItem, ListItemAvatar, ListItemText, Typography } from "@mui/material";
+
+// project imports
+import MainCard from "ui-component/cards/MainCard";
+import TotalIncomeCard from "ui-component/cards/Skeleton/TotalIncomeCard";
+
+// assets
+import StorefrontTwoToneIcon from "@mui/icons-material/StorefrontTwoTone";
+
+// styles
+const CardWrapper = styled(MainCard)(({ theme }) => ({
+    overflow: "hidden",
+    position: "relative",
+    "&:after": {
+        content: '""',
+        position: "absolute",
+        width: 210,
+        height: 210,
+        background: `linear-gradient(210.04deg, ${theme.palette.warning.dark} -50.94%, rgba(144, 202, 249, 0) 83.49%)`,
+        borderRadius: "50%",
+        top: -30,
+        right: -180
+    },
+    "&:before": {
+        content: '""',
+        position: "absolute",
+        width: 210,
+        height: 210,
+        background: `linear-gradient(140.9deg, ${theme.palette.warning.dark} -14.02%, rgba(144, 202, 249, 0) 70.50%)`,
+        borderRadius: "50%",
+        top: -160,
+        right: -130
+    }
+}));
+
+// ==============================|| DASHBOARD - TOTAL INCOME LIGHT CARD ||============================== //
+
+const Group2 = ({ isLoading }) => {
```

Comment:
Changes seem unrelated to FE-12, please move to correct Issue Branch if they're out of scope for this issue